src/main/java/tutorease/address/logic/commands/AddLessonCommand.java

```diff
@@ -62,7 +62,7 @@ public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         ObservableList<Person> personList = model.getFilteredPersonList();
 
-        if (studentId.getValue() > personList.size()) {
+        if (studentId.getValue() >= personList.size()) {
```

Comment:
As the studentId is one-based index, we use >= personList.size() instead of > personList.size()